rules/rules.doc.bzl

```diff
@@ -0,0 +1,9 @@
+"""
+Helper rules for Apple platforms
+"""
+
+load("//rules/private:apple_genrule.bzl", _apple_genrule = "apple_genrule")
+load("//rules:universal_binary.bzl", _universal_binary = "universal_binary")
+
+apple_genrule = _apple_genrule
+universal_binary = _universal_binary
```

Comment:
this file isn't meant to be included but needs to exist to centralize the rules for stardoc so we have a single markdown file for both rules